editor/components/block-list/block.js

```diff
@@ -471,9 +471,12 @@ export class BlockListBlock extends Component {
 		const { onMouseLeave, onReplace } = this.props;
 
 		// Determine whether the block has props to apply to the wrapper.
-		let wrapperProps;
+		let wrapperProps = this.props.wrapperProps;
```

Comment:
@aduth, do I assume correctly that we should be able to remove this if statement as soon as we migrate all blocks to use `supports`?

Reply:
My hope is that we can remove `getEditWrapperProps` at some point, yes.